scripts/validate.py

```diff
@@ -162,9 +166,35 @@ def get_branch_message(branch):
         return None, None
 
 
+def get_branch_message(branch):
+    feature = branch["feature"]["featureId"]
+    if feature == "cfr":
+        value = branch["value"]
+        if "id" in value:
+            return "cfr", branch["value"]
+        return "cfr", None
+    elif feature == "aboutwelcome":
+        value = branch["value"]
```

Comment:
should it be something like feature["value"] instead of branch[value]

Reply:
seeing below error in make check with the patch:

File "scripts/validate.py", line 177, in get_branch_message
    value = branch["value"]